pkg/registry/scheduling/rest/storage_scheduling.go

```diff
@@ -107,12 +107,11 @@ func AddSystemPriorityClasses() genericapiserver.PostStartHookFunc {
 				_, err := schedClientSet.PriorityClasses().Get(pc.Name, metav1.GetOptions{})
 				if err != nil {
 					if apierrors.IsNotFound(err) {
-						// TODO: Remove this explicit conversion after scheduling api move to v1
-						v1beta1PriorityClass := &schedulingv1beta1.PriorityClass{}
-						if err := schedulingapiv1beta1.Convert_scheduling_PriorityClass_To_v1beta1_PriorityClass(pc, v1beta1PriorityClass, nil); err != nil {
+						v1PriorityClass := &schedulingv1.PriorityClass{}
```

Comment:
We still needed conversion because without this `PriorityClasses` were not getting converted to `v1.PriorityClasses` or I have make modifications in other functions(https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/scheduling/helpers.go) to return v1.PriorityClass